src/main/java/core/mvc/JsonView.java

```diff
@@ -1,12 +1,41 @@
 package core.mvc;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.MediaType;
+
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
 import java.util.Map;
 
 public class JsonView implements View {
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
     @Override
     public void render(Map<String, ?> model, HttpServletRequest request, HttpServletResponse response) throws Exception {
+        response.setContentType(MediaType.APPLICATION_JSON_VALUE);
+
+        if (model.isEmpty()) {
+            return;
+        }
+
+        String body = writeValueAsString(model);
+        response.setContentLength(body.getBytes(StandardCharsets.UTF_8).length);
+        response.getWriter().write(body);
+    }
+
+    private String writeValueAsString(Map<String, ?> model) throws JsonProcessingException {
+        if (model.size() == 1) {
+            Object value = model.entrySet()
+                    .stream()
+                    .findFirst()
+                    .get()
+                    .getValue();
```

Comment:
value 만이 필요하므로 아래와 같이 줄일 수도 있겠네요 :)
```suggestion
Object value = model.values().stream()
                    .findFirst()
                    .get()
```

Reply:
앗 ㅋㅋㅋ 어차피 하나만 존재하니까 value에서 바로가져오는게 낫겠네요 감사합니다